### PR TITLE
Added flash algorithm for W7500 ( WIZnet )

### PR DESCRIPTION
**Pull request by kaizen8501**
Added flash algorithm for W7500

### CODE REVIEW

**Comment by c1728p9** [comment]
Hi @kaizen8501 please rebase on top of master rather than merging

**Comment by kaizen8501** [comment]
Hi @c1728p9 . Does it need to do rebase? I think my repository already sync done with master branch and then I modified my code. Please give me advice. Thank you

File: source/wiznet/W7500/FlashDev.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    ONCHIP,                     // Device Type
+    0x00000000,                 // Flash start address
+    0x00020000,                 // Flash total size (128 KB // + 1 kB)
+    0x00000100,                 // Programming Page Size
+    0,                          // Reserved, must be 0
+    0xFF,                       // Initial Content of Erased Memory
+    100,                        // Program Page Timeout 100 mSec
+    3000,                       // Erase Sector Timeout 3000 mSec
+
+    0x00000100, 0x000000,      // Sector Size  128KB ( 512Sectors)
```
Comment (c1728p9) [inline source/wiznet/W7500/FlashDev.c]: You have sector size set to 256 bytes here. Are the comments wrong?

**Comment by c1728p9** [comment]
Hi @kaizen8501, it makes the history much cleaner if you rebase rather than merging master into the branch you are working from. If you can't don't worry about it. A new github option allows the PR to be automatically rebased before it is merged.

**Pull request merged**